src/sentry/models/userreport.py

```diff
@@ -32,14 +32,9 @@ class Meta:
     __repr__ = sane_repr("event_id", "name", "email")
 
     def notify(self):
-        from django.contrib.auth.models import AnonymousUser
-
-        from sentry.api.serializers import UserReportWithGroupSerializer, serialize
         from sentry.tasks.user_report import user_report
 
-        report = serialize(self, AnonymousUser(), UserReportWithGroupSerializer())
-
         user_report.delay(
             project_id=self.project_id,
-            report=report,
+            report=self.id,
```

Comment:
```suggestion
            report_id=self.id,
```

Shouldn't this be `report_id` as `report` is for the dictionary.

Reply:
Indeed it should. Updated!